### ISSUE TITLE
Not running on Mac or RHEL

### ISSUE THREAD

**Issue by bymafmaf** [sirthias/scala-benchmarking-template#10]
I get the following error when I enter `run`:
```
[info] Compiling 3 Scala sources to /Users/onus/my-benchmark/target/scala-2.9.2/classes...
[error] error while loading CharSequence, class file '/Library/Java/JavaVirtualMachines/jdk1.8.0_131.jdk/Contents/Home/jre/lib/rt.jar(java/lang/CharSequence.class)' is broken
[error] (bad constant pool tag 18 at byte 10)
[error] error while loading AnnotatedElement, class file '/Library/Java/JavaVirtualMachines/jdk1.8.0_131.jdk/Contents/Home/jre/lib/rt.jar(java/lang/reflect/AnnotatedElement.class)' is broken
[error] (bad constant pool tag 18 at byte 76)
[error] two errors found
[error] (compile:compile) Compilation failed
[error] Total time: 2 s, completed Jul 18, 2017 10:15:41 PM
```
Java version:
java version "1.8.0_131"
Java(TM) SE Runtime Environment (build 1.8.0_131-b11)
Java HotSpot(TM) 64-Bit Server VM (build 25.131-b11, mixed mode)

**Comment by sirthias** [comment]
This template is terribly old and not recommended for use anymore.
I'll suggest you turn to @ktosos's [sbt-jmh](https://github.com/ktoso/sbt-jmh) instead.

**Comment by ktoso** [comment]
Would you be ok with adding such note in the readme in case other people end up here still?
(Can PR a quick sentence)

**Comment by sirthias** [comment]
@ktoso Yeah, adding a sentence probably makes sense.
I'll happily merge anything you propose...

**Comment by ktoso** [comment]
Ok done :) maybe too wordy... let me know
